src/components/schemas/project.yaml

```diff
@@ -99,7 +99,43 @@ project_in:
     restricted_spender_user_ids:
       $ref: './fields.yaml#/restricted_spender_user_ids'
     approver_user_ids:
-      $ref: './fields.yaml#/approver_user_ids'
+      allOf:
+        - $ref: './fields.yaml#/approver_user_ids'
+      description: |
+        - List of IDs of users who are approvers of the project.
+        - Either `approver_user_ids` or `approver_emails` can be used but not both.
+        - The default value for approver_user_ids is `[]` and it is applicable only when creating the project.
+    approver_emails:
+      type: array
+      maxItems: 2
+      nullable: False
+      items:
```

Comment:
I have not re-used the `email` definition from the fields.yaml file because there is some bug in openapi library where the `nullable` property cannot be overriden.

TLDR version: `allOf` and `nullable` doesn't work together